conf/README.md

```diff
@@ -81,12 +81,18 @@ from ocs import defaults
 
 # From you code you can access those data like
 
-Taking data from ENV_DATA will always use right cluster_namespace passed via
-`--ocsci-conf` config file or default one defined in `default_config.yaml`.
-function_which_taking_namespace(namespace=config.ENV_DATA['cluster_namespace'])
+# Taking data from ENV_DATA will always use right cluster_namespace passed via
+#`--ocsci-conf` or `--cluster-conf` config files.
+
+# It's not possible to access config data from module level as config is not
```

Comment:
I would reword since its conveying different meaning, maybe just "config data should not be accessed at module level and instead should be used within function definitions or fixtures"

Reply:
done in next commit